.github/workflows/invalidations.yml

```diff
@@ -0,0 +1,33 @@
+name: Invalidations
+on: pull_request
```

Comment:
Maybe this should be run only for PRs to master as it compares invalidations with master?

Reply:
@ranocha

Reply:
I used the setup from https://github.com/SciML/MuladdMacro.jl/pull/26 and started MassInstallAction.jl, so it's too late to cancel it right now. But since someone else needs to run it again - or I need to get more rights in SciML - we can change that afterwards. Please make a PR with the change you would like to have to https://github.com/SciML/MuladdMacro.jl and we can use it as a template for the future process of adding it to every SciML repo.